Lib/test/test_base64.py

```diff
@@ -394,10 +399,6 @@ def test_b32decode_casefold(self):
         self.assertRaises(binascii.Error, base64.b32decode, b'me======')
         self.assertRaises(binascii.Error, base64.b32decode, 'me======')
 
-        # Mapping zero and one
```

Comment:
Is there a reason these tests are removed? I didn't see a mention of `b32decode` behavior changes in the changelog.

Reply:
Do not mind. This is a clean up after other change. I already have cleaned up this in other PR.